### ISSUE TITLE
[UI idea] Improve language dropdown

### ISSUE THREAD

**Issue by arjank** [Felienne/hedy#2917]
**Language dropdown**
The language dropdown is hard to find if you do not speak the selected language, due to lack of an identifying item.

I also noticed that the dropdown is not keyboard selectable.

**Comment by arjank** [comment]
The first part has been discussed with @Felienne already, and I'll work on this issue.

**Comment by Felienne** [comment]
Thanks for noticing both @arjank!! 

Keyboard input it an important one to test for all menus (great inspiration yesterday by Leonie, or course!) Maybe we can make a separate issue for more buttons or was this the only one that did not work?

About the icon: Do you have a user account? Because if you login there is a new dropdown where we do have icons in the dropdown, so maybe we can make the globe look a bit similar:

<img width="242" alt="image" src="https://user-images.githubusercontent.com/1003685/174428520-de273573-b396-4e92-a8d2-134fefa63e14.png">

**Comment by arjank** [comment]
There are multiple improvements for the UI, and I would suggest multiple tickets for that. Lots of small steps allow for more progress.

**Comment by Felienne** [comment]
> There are multiple improvements for the UI, and I would suggest multiple tickets for that. Lots of small steps allow for more progress.

Yes separate issues is easier to keep track of things. Feel free to open issues for these UI things (especially if you plan to also make pull requests for them haha)! @TiBiBa is mostly working on the front end, so if you want to discuss you plans before starting, he is a good person to contact.
